narayana-spring-boot-core/src/main/java/me/snowdrop/boot/narayana/core/jms/GenericXAConnectionFactoryWrapper.java

```diff
@@ -32,18 +33,31 @@
     /**
      * Create a new {@link GenericXAConnectionFactoryWrapper} instance.
      *
-     * @param transactionManager underlying transaction manager
-     * @param xaRecoveryModule   recovery module to register data source with.
-     * @param properties         Narayana properties
+     * @param transactionManager  underlying transaction manager
+     * @param xaRecoveryModule    recovery module to register data source with.
+     * @param properties          MessagingHub properties
      */
     public GenericXAConnectionFactoryWrapper(TransactionManager transactionManager, XARecoveryModule xaRecoveryModule,
-            NarayanaProperties properties) {
-        super(xaRecoveryModule, properties);
+            MessagingHubConnectionFactoryProperties properties) {
```

Comment:
Pooled wrapper properties should not be here

Reply:
Right, I'll remove it.